Disables Take a Photo Button and File Chooser While Camera is Active

Ryan10145 opened the pull request:
Fixes #1321

This disables the other two buttons after the "Take a Photo" button is pressed and reenables them once the camera is closed.

Before "Take a Photo" is pressed and after camera is closed:
![Screen Shot 2019-12-13 at 9 04 17 PM](https://user-images.githubusercontent.com/8119928/70841998-2006e980-1ded-11ea-95ca-e63038b832ed.png)

After "Take a Photo" is pressed (I covered the webcam with my finger which is why the camera stream appears all black):
![Screen Shot 2019-12-13 at 9 04 28 PM](https://user-images.githubusercontent.com/8119928/70842009-2bf2ab80-1ded-11ea-9696-1f263904d727.png)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [X] code is in uniquely-named feature branch and has no merge conflicts
* [X] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below
* [X] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

Ryan10145 commented:
@publiclab/is-reviewers Please review this, thank you!

keshav234156 commented:
@Ryan10145 Can you please  upload a gif of the changes made. After the change "click picture" and "close" buttons were not working for me and also the camera was not able not.

Ryan10145 commented:
@keshav234156 I'm not sure why that's happening, as it works fine on my machine, as shown in the GIF below. Can you check to make sure that the webcam works properly before the changes? Even before my changes, I had an issue before where on my computer without a webcam, the webcam stream wouldn't show (because it didn't exist) and also the buttons wouldn't work, which sounds similar to your situation. To test and incorporate these changes, I switched to a laptop which did have a webcam. I was going to further investigate and if necessary, create an issue/pull request to fix this problem.

![Dec-14-2019 10-49-48](https://user-images.githubusercontent.com/8119928/70851167-e9b68200-1e5f-11ea-9984-8f288573ee59.gif)
I blocked the webcam again with my thumb, although I moved it back and forth to show that it is a live video.

keshav234156 commented on src/modules/ImportImage/Ui.js:
```diff
@@ -9,7 +9,7 @@ module.exports = function ImportImageModuleUi(step, ui) {
     var dropZone = '\
     <div class="dropzone import-image-zone" id="' + dropzoneId + '">\
-          <p>\
+          <p id="dropzone-text">\
```
I think this change was not needed.

Ryan10145 replied:
Sorry about that, that was one of my testing changes that I forgot to remove before committing. You're correct, that change was not needed. Thanks for pointing that out!

jywarren commented:
Thanks, all!!! 🎉
